core/sys/darwin/Security/SecBase.odin

```diff
@@ -0,0 +1,386 @@
+package Security
+
+OSStatus :: distinct i32
+
+errSec :: enum OSStatus {
+	Success = 0, // No error.
+	Unimplemented = -4, // Function or operation not implemented.
+	errSecDiskFull = -34, // The disk is full.
```

Comment:
I'd probably remove all of the `errSec` prefixes.

Reply:
> I'd probably remove all of the `errSec` prefixes.

Good point, thanks.